### PR TITLE
Added support for LibMan and removed old banner images

### PR DESCRIPTION
**Pull request by JustCallMeAD**
Modified the starter template so that it supports LibMan by default. Also deleted the old banner images as they are not being used anymore.

### CODE REVIEW

**Comment by JustCallMeAD** [comment]
NOTE: Will need to make similar changes to the identity UI templates. I can do a pull request for these changes too if you would like me too?

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/libman.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "version": "1.0",
+  "defaultProvider": "cdnjs",
+  "libraries": [
+    {
+      "provider": "cdnjs",
```
Comment (scottaddie) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/libman.json]: You can remove the `provider` property from each of these object literals. The `defaultProvider` property will be used.

**Comment by scottaddie** [comment]
@justcla You may be interested in this PR.

**Comment by Eilon** [comment]
@mkArtakMSFT / @ryanbrandenburg - please check with @DamianEdwards on when/how we should get this in.

**Comment by ryanbrandenburg** [comment]
[Seems like](https://github.com/aspnet/Templating/issues/471#event-1683852954) we're targeting LibMan for 3.0.0-preview1.

**Comment by ryanbrandenburg** [comment]
Please rebase onto the latest from master. I ended up removing the banner images independently and making other changes in the mean time and there's a conflict in StarterWeb's _Layout.cshtml, so it will be much easier to see what's actually changing once this PR has the latest.

**Comment by ryanbrandenburg** [comment]
@JustCallMeAD, a reminder to rebase this onto master so we can more easily see what the important changes are.

**Review by ryanbrandenburg — CHANGES_REQUESTED** [review]
I poked around at this locally and I think there are some extra things we'll want to address when doing this:

1. We should ensure that LibMan is installed, then run `libman restore` before the build step. A target similar to https://github.com/aspnet/Templating/blob/master/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in#L40 would do the trick.
2. Let's not check the files that LibMan pulls down for us into Git anymore (they'll just get overridden anyway). That will break our baseline tests, so we'll have to update those and also confirm that all scenarios still work, integrity attributes still apply etc.

Since the LibMan work is scheduled for 3.0 (and we're still doing 2.2 work) either you can do those cleanup items or it'll be a bit of a wait before I can carve out some time. Either way this is a pretty good start and probably what we'll be looking to for #471.

**Comment by Eilon** [comment]
Hi, we are closing this PR because the source code has all been moved to https://github.com/aspnet/AspNetCore/tree/master/src/Templating so that all the ASP.NET Core source code is in one place. If you are interested in continuing this contribution, we recommend you log a new issue in https://github.com/aspnet/AspNetCore/issues with details and if the proposal is accepted, send a new PR so the new repo. Sorry for any inconvenience!

**Pull request closed without merging**